galaxy_ie_helpers/__init__.py

```diff
@@ -201,8 +201,9 @@ def get(datasets_identifiers, identifier_type='hid', history_id=None, retrieve_d
 
     ## First path if only one item given, otherwise all paths.
     ## Should not break compatibility.
-    if retrieve_datatype == True:
-        return file_path_all[0] if len(file_path_all) == 1 else file_path_all, datatypes_all[0] if len(file_path_all) == 1 else datatypes_all
+    if retrieve_datatype:
+        return file_path_all[0] if len(file_path_all) == 1  \
```

Comment:
What I mean is something like:

 if len(file_path_all) == 1 else datatypes_all:
    if len(file_path_all) == 1 
        return file_path_all[0] 
    else:
           return file_path_all, datatypes_all[0]